### ISSUE TITLE
Remove exploration from viz rollout

### ISSUE THREAD

**Issue by michaelriedl** [michaelriedl/2048-ppo-agent#46]
Currently the visualization rollout still samples the action space based on the logits. Instead, this should take the argmax after a softmax.
